### ISSUE TITLE
Play nicely with Ulaunchers gi dependancies

### ISSUE THREAD

**Issue by tjquillan** [iboyperson/ulauncher-system#14]
As shown in the PR https://github.com/Ulauncher/Ulauncher/pull/717 Ulauncher does not currently explicitly declare all of its gi dependency versions. As @feimosi mentioned (https://github.com/iboyperson/ulauncher-system/issues/10#issuecomment-818504501) this appears to be causing issues for systems running Gnome 40.

While this is a Ulauncher problem I will see what can be done to fix this in the short run.

Related Ulauncher issue: https://github.com/Ulauncher/Ulauncher/issues/703

Error:
```
2021-04-13 09:15:35,246 | ERROR | ulauncher.api.server.ExtensionRunner: _run_process() | Extension "com.github.iboyperson.ulauncher-system" exited instantly with code 1
2021-04-13 09:15:35,246 | ERROR | ulauncher.api.server.ExtensionRunner: _run_process() | Extension "com.github.iboyperson.ulauncher-system" failed with an error: ImportError: Requiring namespace 'Gdk' version '3.0', but '4.0' is already loaded
```

**Comment by tjquillan** [comment]
@feimosi Can you verify if this is fixed? I pushed 1d77680c75b2996be0df9ca710669b0c2d668758 which will hopefully address it  but I do not have a gnome 40 system to test on.

**Comment by feimosi** [comment]
Unfortunately not, the same error. Hopefully Ulauncher PR gets merged soon.

**Comment by tjquillan** [comment]
Closing this as I am not sure if I can do anything to resolve this.

**Comment by tjquillan** [comment]
This is now properly fixed by 639facd8be9208afde0371def31913b0804709e8

**Comment by feimosi** [comment]
Great, thanks!
